bin/hbase-daemon.sh

```diff
@@ -155,12 +155,20 @@ JAVA=$JAVA_HOME/bin/java
 export HBASE_LOG_PREFIX=hbase-$HBASE_IDENT_STRING-$command-$HOSTNAME
 export HBASE_LOGFILE=$HBASE_LOG_PREFIX.log
 
-if [ -z "${HBASE_ROOT_LOGGER}" ]; then
```

Comment:
If provided this string we could munge this string into LOGGER_APPENDER and LOGGER_LEVEL and warn about the change in environment variable usage for additional compatibility.

Reply:
Ah, now we do not need HBASE_ROOT_LOGGER_LEVEL after log4j2 supports set lvel and appenders at once. I only changed the hbase script back and missed this hbase-daemon script. Let me fix.